tools/minishift-build/rest.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+. $(dirname `realpath $0`)/vars.sh
+
+prepare_dir syndesis-rest
+mvn clean install -Pflash
+cd runtime
+mvn clean package fabric8:build -Dfabric8.mode=kubernetes -Pflash
```

Comment:
yes, could be. Haven't checked, just put my workflow into some script form :) I will try ...